alphastats/DataSet.py

```diff
@@ -9,17 +9,21 @@
 from alphastats.loader.FragPipeLoader import FragPipeLoader
 from alphastats.loader.MaxQuantLoader import MaxQuantLoader
 import os
-import plotly.express as px
-import scipy.stats
-import dash_bio
 import numpy as np
 import logging
-import sys
 from sklearn_pandas import DataFrameMapper
-import sklearn
+import warnings
+from alphastats._DataSet_Plot import Plot
```

Comment:
As my DataSet script with all functions for plotting, and preprocessing would become quite large and messy, I decided to split it up into several files and used this suggestion (last section): http://www.qtrac.eu/pyclassmulti.html

My (not very solid) reasoning for the underscore, is that they also used an underscore. My intention was to use the class Plot from "_DataSet_Plot"  internally to inherit the functions to DataSet and not as a standalone class. 

But I think it will make more sense to remove the underscore.